taccsite_cms/static/site_cms/css/src/_imports/trumps/s-document.css

```diff
@@ -93,6 +93,12 @@ p + dl.small {
 
   border: var(--global-border--normal);
 }
+.s-document i.icon--as-image {
+  font-size: 2.8em; /* body line-height * 2 */
+  display: inline-block;
+  padding: 0.25em;
+  border: var(--global-border--normal);
+}
```

Comment:
<details><summary>An <code>.s-</code> what now? No where more official to style icons?</summary>

The `.s-` namespace is a "scope", which is a kind of "trump" ([ITCSS methodology](https://confluence.tacc.utexas.edu/x/IAA9Cw)). Trump styles are allowed to style other components and generally overwrite whatever; I don't want to support this icon image thing everywhere, because it's not part of any official design. This specific stylesheet is just for the guide pages, which still may be temporary (it may be moved to a TACC docs system for TUP redesign, a docs system that Alex is building infrastructure to support).

</details>